### ISSUE TITLE
problem of rendering 

### ISSUE THREAD

**Issue by angchen-dev** [ForMyCat/SparseGS#6]
Dear authors, thank you so much for sharing your work! 

I tried to run the code on the garden scene with 12 views without floater pruning, as described in the ablation section of your paper. Unfortunately, in my environment, the rendered RGB images have all zero values and the depth maps show values of [0, 0, 127, 255]. Perhaps I missed something. Could you please provide some suggestions?

Rendered RGB image:
![DSC07956](https://github.com/ForMyCat/SparseGS/assets/8221022/cf314f31-50bd-40a1-9422-5e2d991feb60)

Rendered Depth map:
![DSC07956_depth](https://github.com/ForMyCat/SparseGS/assets/8221022/181eb456-bb10-411c-8c9a-9ff3aff19a66)

Here are the details of my environment:

Ubuntu OS: 22.04
Python: 3.10.14
CUDA: 11.8
PyTorch: 2.2.2
Torchvision: 0.17.2
Torchaudio: 2.2.2

Command:
```
# training
python3 train.py --source_path ./data/12_view/garden_12 --model_path ./output/garden_12_SparseGS --beta 5.0 --lambda_pearson 0.05 --lambda_local_pearson 0.15 --box_p 128 --p_corr 0.5 --lambda_diffusion 0.001 --step_ratio 0.99 --lambda_reg 0.1 --iterations 30000 --checkpoint_iterations 30000 -r 2

# render
python3 render.py --source_path ./data/full_view/garden --model_path ./output/garden_12_SparseGS --no_load_depth --iteration 30000

# evaluation
python3 metrics.py --model_path ./output/garden_12_SparseGS --exclude_path ./data/12_view/garden_12
```

Thank you in advance for your help. :)

**Comment by ForMyCat** [comment]
Sorry I am not able to replicate your result. Can you try one component at a time? Start with only global depth loss and see whether the renderings are still incorrect. If training garden takes too long, you can debug with bonsai.

**Comment by angchen-dev** [comment]
Thank you for your suggestions. I solved the problem by using CUDA 12
